app/assets/stylesheets/pages/home.scss

```diff
@@ -73,18 +112,47 @@ $small-screen: em(420);
 
 
 #home-title {
-  padding-top: 50px;
-
+  padding-top: 75px;
   @include media(max-width $small-screen) {
-    padding-top: 30px;
+    padding-top: 32px;
   }
 }
 
+#home-subtitle {
+  padding-top: 5px;
+
+  @include media(max-width $medium-screen) {
+    padding-top: 25px;
+  }
+}
+
+.bold-blue {
+  font-weight: bold;
+  color: #477DCA;
```

Comment:
Color literals like `#477DCA` should only be used in variable declarations; they should be referred to via variable everywhere else.